README.md

```diff
@@ -72,7 +72,13 @@ Set up the database:
     rails db:schema:load
     rails r db/scripts/create_tags_path_view.rb
     rails db:migrate
-    rails db:seed
+
+&nbsp;
+
+    $ rails db:seed
+    Category: Created 2, skipped 0
+    [...]
+    $ _
```

Comment:
What's this line for?

Reply:
I represent a blinking prompt with the underscore character (a caret would be better...), but it doesn't make sense if the last lines aren't shown. I should remove it.